include/RAJA/policy/cuda/kernel/CudaKernel.hpp

```diff
@@ -649,13 +657,25 @@ struct StatementExecutor<
       {
         auto func = launch_t::get_func();
 
+        using EXEC_POL =
+            ::RAJA::policy::cuda::cuda_exec_explicit<LaunchConfig, void, void,
+                                                     0, true>;
+
+        RAJA::cuda::detail::cudaInfo launch_info;
+        launch_info.gridDim      = launch_dims.dims.blocks;
+        launch_info.blockDim     = launch_dims.dims.threads;
+        launch_info.dynamic_smem = &shmem;
+        launch_info.res          = res;
         //
         // Privatize the LoopData, using make_launch_body to setup reductions
         //
         // Note that there is a circular dependency between the previous setup
         // of the launch_dims and potential changes to shmem here that is
         // currently an unresolved issue.
         //
+        auto reducer_tuple = RAJA::expt::filter_reducers(data.param_tuple);
```

Comment:
Also the comment above this really goes with the make_launch_body call so I would move this code above that.